src/pages/docs/octopus-rest-api/cli/octopus-deployment-target-kubernetes-create.mdx

```diff
@@ -27,8 +27,8 @@ Flags:
       --certificate-path string                 The path to the CA certificate of the cluster. The default value usually is: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
       --client-certificate string               Name of client certificate in Octopus Deploy
       --cluster-url string                      Kubernetes cluster URL. Must be an absolute URL. e.g. https://kubernetes.example.com
-      --docker-container-registry string        The feed of the docker container registery to use if running health check in a container on the worker
-      --docker-image-flags string               The image (including the tag) to use from the container registery
+      --docker-container-registry string        The feed of the docker container registry to use if running health check in a container on the worker
+      --docker-image-flags string               The image (including the tag) to use from the container registry
```

Comment:
This is probably based on an actual program's buggy output, so please consider this as a bug report.